pkgs/by-name/li/libmsquic/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  lttng-tools,
+  libatomic_ops,
+  perl,
+  coreutils,
+  ...
```

Comment:
Why `...` but no `@` on the attr set? This seems to be not necessary.

Reply:
Sorry, its been a while since I packaged for nixpkgs. Fixing.